src/components/userContent/cardTemplate/card.js

```diff
@@ -0,0 +1,63 @@
+import './card.css';
+import card from './card.hbs';
+import Header from './cardHeader/header';
+import OrderItem from './orderItem/orderItem';
+import ShowMore from './showMore/showMore';
+
+
+export default class Card {
+  render(target, props) {
+    let orderPrice = 0;
+    for (const order of props.orders) {
+      orderPrice += order.price;
+    }
+
+    const cardContext = {
+      orderPrice,
+      active: props.header.active,
+      button: props.orders.length > 0 ? 'Редактировать' : 'Заказать',
+      sumPrice: props.orders.length > 0,
+    };
+
+    const cardTemplate = card(cardContext);
+    target.innerHTML = cardTemplate;
+
+    if (props.header.active) {
+      props.header.headerStyle = 'active-card';
+    } else {
+      props.header.headerStyle = 'inactive-card';
+    }
+
+    const header = new Header();
+    header.render(target.querySelector('.header'), props.header);
+
+    if (!props.header.active) {
+      const cardContent = target.querySelector('.card-content');
+      cardContent.innerText = 'Меню еще недоступно';
+      target.querySelector('.card-inside').classList.add('.not-available-menu');
```

Comment:
Please do this in .hbs file. Keep visual logic in templates and create props in JS.